mobile/src/constants/links.ts

```diff
@@ -1,5 +1,5 @@
-import { Linking } from 'expo';
-import { Alert } from 'react-native';
+//import { Linking } from 'expo';
+import { Alert, Linking } from 'react-native';
```

Comment:
Is there a reason for this?

Reply:
I didn't change this file actually, I guess it was modified by the PR I merged

Reply:
Should remove it if it isn't used anywhere in the code. In either case, the commented import above should be removed as well.